src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,39 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+import java.util.Arrays;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        int userScore = 0;
```

Comment:
this solution is over complicated, try to make it simpler
- firstly we can split `record` by ":" and save the result to the array.
- then we can compare the first value from the array with the email
- if comparing return true - you should return the second value from the array.

Reply:
done!